server/routes/routes.js

```diff
@@ -9,6 +9,7 @@ const router = express.Router();
 
 // const mongoose = require('mongoose');
 // const MongoStore = require('connect-mongo');
+// const sighting = require('../lib/sightingsController.js');
```

Comment:
Clean commented code up please

Reply:
Will mongoose and sighting (commented lines 10-32) be required once everything is active?  I've cleaned up the bottom of the file now that the API calls aren't in routes.js

Reply:
No, I don't think we should be doing any db calls in the router

Reply:
Keeping session/user sections for now, but deleted the mongoose specific sections.